docs/source/install/tools.rst

```diff
@@ -0,0 +1,91 @@
+==================
+Tools Installation
+==================
+
+Overview
+********
+
+This document describes how to install following PDS Registry tools (command-line applications):
+
+ * **Standalone Harvest** extracts metadata from PDS4 labels and loads extracted metadata into PDS Registry.
+ * **Harvest Client** has to be used with Scalable Harvest server components to submit asynchronous jobs
```

Comment:
Should we use the term "**Harvest Client CLI**" (instead of "**Harvest Client**") here to be consistent with the "**Harvest Client CLI**"  term used in the diagram at docs/source/_static/images/registry-harvest.png?

Reply:
I don't know. Let's discuss.